README.md

```diff
@@ -7,9 +7,91 @@ Table of Contents
 =================
 
 * [Name](#name)
+* [Synopsis](#synopsis)
 * [Description](#description)
 * [Copyright & License](#copyright--license)
 
+Synopsis
+========
+
+Basic usage
+
+```
+
+# assignment in opsboy file scope.
+foo = bar; # Opsboy's assignment syntax. variable can be interpolated when startwith dollar sign and within strings.
+
+# comment in opsboy file.
+#foo = bar; # opsboy use # as comment syntax.
+
+# rule and target.
+
+Basically,
+
+    foo {
+	directive1;
+	directive2;
+    }
+
+is a rule. And foo is the target. It's quite similar to Makefile's rules perse.
+
+We use Opsboy's compile out perl and send make and target arguments to execute 
+it.
+
+./openresty-test.pl make foo 
+
+# command
+All opsboy command will list below.
```

Comment:
This sentence has English grammar errors. Should be

"All opsboy syntactic structures and directives are listed below"